### DISCUSSION TITLE
Can you support the official leg tracker PICO

### DISCUSSION THREAD

**Discussion by 755713372** [alvr-org/ALVR#1998] [General]
The official streaming function of leg tracking has not been updated. Now the official streaming is very difficult to use and often crashes and gets stuck. Can you support this function?

**Comment by zmerp** [comment]
I don't think Pico exposes any leg or body tracking extension for OpenXR yet, so we cannot support it. Same goes for face tracking.

**Comment by app-gitKaiwho** [comment]
Early September they released the V3.0.0 of their SDK with AR foundation support. Wouldn't this mean Pico Motion tracker support is now feasible ?

https://developer.picoxr.com/document/updates-unity/release-notes-for-version-300/

**Reply by zmerp** [reply]
We use the OpenXR SDK which hasn't changed since June

**Reply by app-gitKaiwho** [reply]
well to bad, thanks for your time
